myp5.js

```diff
@@ -1,6 +1,6 @@
 
 
-var lyrics = [];
+var lyricObjects = [];
 
```

Comment:
You're using the variable `lyrics` twice, both for the text data and the array of "Lyric" objects you create, causing a lot of confusion in your code. It helps to name your variable very explicitly.